test_scripts/Defects/8_2/3931_Generating_vrHelp_and_helpPrompt_after_ResetGlobalProperties.lua

```diff
@@ -0,0 +1,167 @@
+  helpPrompt = {
+    {text = "Please speak one of the following commands,", type = "TEXT"},
+    {text = "Please say a command,", type = "TEXT"}
+  } 
+}
+
+--[[ Local Functions ]]
+local function AddCommand(pParams)
+  local cid = common.getMobileSession():SendRPC("AddCommand", pParams)
+  common.getHMIConnection():ExpectRequest("UI.AddCommand", {
+    cmdID = pParams.cmdID,
+    menuParams = pParams.menuParams
+  })
+  :Do(function(_, data)
+    common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+  end)
+  common.getHMIConnection():ExpectRequest("VR.AddCommand", {
+    cmdID = pParams.cmdID,
+    vrCommands = pParams.vrCommands
+  })
+  :Do(function(_, data)
+    common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+  end)
+
+  table.insert(generatedGlobalProperties.vrHelp, { position = (#generatedGlobalProperties.vrHelp + 1), text = pParams.vrCommands[1] })
+  table.insert(generatedGlobalProperties.helpPrompt, { text = pParams.vrCommands[1], type = "TEXT" })
+  common.getHMIConnection():ExpectRequest("UI.SetGlobalProperties", {
+    vrHelp = generatedGlobalProperties.vrHelp
+  })
+  :Do(function(_, data)
+    common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+  end)
+  common.getHMIConnection():ExpectRequest("TTS.SetGlobalProperties", {
+    helpPrompt = generatedGlobalProperties.helpPrompt
+  })
+  :Do(function(_, data)
+    common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+  end)
+
+  common.getMobileSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+end
+
+local function SetGlobalProperties(pParams)
+  local cid = common.getMobileSession():SendRPC("SetGlobalProperties", pParams)
+  common.getHMIConnection():ExpectRequest("UI.SetGlobalProperties", {
+    vrHelp = pParams.vrHelp,
+    vrHelpTitle = pParams.vrHelpTitle
+  })
+  :Do(function(_, data)
+    common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+  end)
+  common.getHMIConnection():ExpectRequest("TTS.SetGlobalProperties", {
+    helpPrompt = pParams.helpPrompt
+  })
+  :Do(function(_, data)
+    common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+  end)
+  common.getMobileSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+
+end
+
+local function ResetGlobalProperties(pParams, defaultParams)
+  local cid = common.getMobileSession():SendRPC("ResetGlobalProperties", pParams)
+  common.getHMIConnection():ExpectRequest("UI.SetGlobalProperties", {
+    vrHelp = defaultParams.vrHelp,
+    vrHelpTitle = defaultParams.vrHelpTitle
+  })
+  :Do(function(_, data)
+    common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+  end)
+  common.getHMIConnection():ExpectRequest("TTS.SetGlobalProperties", {
+    helpPrompt = defaultParams.helpPrompt
+  })
+  :Do(function(_, data)
+    common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+  end)
```

Comment:
```suggestion
  common.getHMIConnection():ExpectRequest("UI.SetGlobalProperties", {
    vrHelp = generatedGlobalProperties.vrHelp
  })
  :Do(function(_, data)
    common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
  end)
  common.getHMIConnection():ExpectRequest("TTS.SetGlobalProperties", {
    helpPrompt = generatedGlobalProperties.helpPrompt
  })
  :Do(function(_, data)
    common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
  end)
```
These requests should match up with the original generated values.